marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getUris.api

```diff
@@ -0,0 +1,17 @@
+{
+    "functionName": "getUris",
+    "params": [
+        {
+           "name": "stepName",
+           "datatype": "string"
+        },
+        {
+           "name": "limit",
```

Comment:
I think this would benefit from a "desc" stating what the limit is if no value is provided

Reply:
And make this "int" or "integer", whichever DS prefers.